doc/api/url.md

```diff
@@ -777,6 +777,21 @@ Returns an ES6 Iterator over the names of each name-value pair.
 Remove any existing name-value pairs whose name is `name` and append a new
 name-value pair.
 
+#### urlSearchParams.sort()
+
+Sort all existing name-value pairs in-place by their names. Sorting is done
+with a [stable sorting algorithm][], so relative order between name-value pairs
+with the same name is preserved.
+
+This method can be used, in particular, to increase cache hits.
+
+```js
+const params = new URLSearchParams('query[]=123&type=search&query[]=abc');
+params.sort();
+console.log(params.toString());
+  // Prints query%5B%5D=123&query%5B%5D=abc&type=search
+```
```

Comment:
It may be better to swap the `query[]=123` and `query[]=abc` positions to show that the sort will not affect the ordering of the `abc` and `123` positions given that `123` would typically appear before `abc`